packages/react-scheduler/src/ReactScheduler.js

```diff
@@ -123,21 +122,21 @@ if (!canUseDOM) {
     localClearTimeout(timeoutId);
   };
 } else {
-  if (typeof localRequestAnimationFrame !== 'function') {
-    warningWithoutStack(
-      false,
-      "This browser doesn't support requestAnimationFrame. " +
-        'Make sure that you load a ' +
-        'polyfill in older browsers. https://fb.me/react-polyfills',
-    );
-  }
-  if (typeof localCancelAnimationFrame !== 'function') {
-    warningWithoutStack(
-      false,
-      "This browser doesn't support cancelAnimationFrame. " +
-        'Make sure that you load a ' +
-        'polyfill in older browsers. https://fb.me/react-polyfills',
-    );
+  if (__DEV__ && typeof console !== 'undefined') {
```

Comment:
Nit: we tend to put DEV check in its own separate if block, and then nest another if (when necessary). Makes DEV conditions stand out more